### DISCUSSION TITLE
Please mention capacitor properities

### DISCUSSION THREAD

**Discussion by UweBonnes** [ozel/DIY_particle_detector#14] [Q&A] [answered]
Hello,

without hints in the BOM, people will tend to buy Y or even Z type ceramic capacitors at least for the high capacitance values , 100 n C3 and C7 and  470 n C4. Y/Z are know for microphonic and other bad effect, and at least  C3, C7 and C4 should be of X or even COG quality. 100 nF COG is not cheap, but achievable. With C4 -> 100 n, R4 > 22k, R5 -> 4M7 and C6 -> 680 p, all C3, C7 and C7 could be specified as 100 nF/COG. Otherwise bothe bias dividers R1,R2,R6 and R7 could be made 10 time higher without bad influence but extend battery lifetime.

Please at least state in the BOM that Y/Z ceramics is deprecated and COG for C3/C7 is preferred.

**Comment by ozel** [comment, marked as answer]
Hi Uwe,

thanks for your comments related to the alpha-spectrometer variant.
On a first note, I guess you must have missed the bom folder. The two .csv files there feature specific parts with order codes for the suppliers listed on the related kitspace site of the project: 
https://kitspace.org/boards/github.com/ozel/diy_particle_detector/alpha-spectrometer/ 

For the large C4, I actually went with a Z type. C1 and C2 are of course C0G, but for the others (selected X and Y types), I doubt that it matters enough in terms of SNR performance and variances in the energy response. However, I am happy to reconsider if you have noticed improvements with other choices - please let me know!

Since the alpha-spectrometer variant of the circuit does actually exhibit quite some microphonic effect, I had replaced all capacitors with C0G types as a test. But since that did not reduce the microphonics, I concluded the PIN diode itself must be coupling capacitively with the soft tin metal walls which in turn pick up vibrations easily (this does not happen with thick-walled aluminum die-cast cases as explained in the Wiki).
I would be curious to know if you did a similar capacitor comparison in this circuit and noticed improvements.
Did you run into problems with reusing my energy calibration as described in the corresponding paper and the Wiki?

Regarding the general choice of parts: This project is really geared towards electronic beginners and high-school level physics teaching environments where people are often not experienced tinkerers. Therefore I tried to keep the overall amount of parts as well as the number of different resistor/capacitor values as low as possible.

For a detector signal processing expert, the choice of time constants after the first stage may look unusual. But they are on purpose in order to decrease the rise time of the first edge as well as keeping the complete pulse as bipolar as possible while maintaining large enough SNR and pulse width (at an overall low number of components). 
Without the extra slowing down, widening, and 'bi-polarisation' of the pulse, certain headset sockets will detect it as an unexpected glitch and switch off the sound input. The circuit is trying to emulate a microphone signal after all which can be sampled well at 48 kHz.

Regarding the bias resistors, I thought about increasing them but eventually kept them low in favor of reducing the number of different resistors (10k and 4k7 are reused). Their fraction of the current consumption is only 20%, the majority is consumed by the opamps. In general, I am recommending to use good old 9V NIMH accumulators, c.f. the Wiki.
